src/makeCalculator.js

```diff
@@ -1,10 +1,36 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate: (callback, number) => {
+      callback(number);
```

Comment:
The `operate` method should update the `result` property based on the callback's operation. Currently, it calls the callback but does not modify the `result`. Ensure that the `result` is updated appropriately after the callback is executed.